python/hail/dataset.py

```diff
@@ -19,13 +17,20 @@
 
 
 @decorator
+def requireSampleTString(func, vds, *args, **kwargs):
```

Comment:
shouldn't this go on the scala side? it doesn't do any conversion, which is why requireTGenotype is on the py side.